doc/tutorials/music_tutorial/music_tutorial_1.rst

```diff
@@ -47,18 +47,16 @@ constructs.
 While the focus here is on MUSIC, we need to know a few things about how
 NEST works in order to understand how MUSIC interacts with it.
 
-Go straight to part 1 of tutorial - connect 2 simulations using PyNEST
-
 The Basics of NEST
-~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~
 
 A NEST network consists of three types of elements: neurons, devices,
 and connections between them.
 
 Neurons are the basic building blocks, and in NEST they are generally
 spiking point neuron models. Devices are supporting units that for
 instance generate inputs to neurons or record data from them. The
-Poisson spike generator, the spike detector recording device and the
+poisson spike generator, the spike detector recording device, and the
```

Comment:
```suggestion
Poisson spike generator, the spike detector recording device, and the
```

The distribution is called after a french mathematician, so the name is written with upper-case starting letter. So either write it like it's supposed to be , or say `poisson_generator` and `spike_detector` (probably both in double backticks to get them formatted and linked properly). I'm fine with both and provide one of them for a seamless clicking review response experience ;-)